verilog/tools/lint/verilog_lint.cc

```diff
@@ -92,7 +92,10 @@ ABSL_FLAG(
     "If true, print the description of every rule formatted for the "
     "Markdown and exit immediately. Intended for the output to be written "
     "to a snippet of Markdown.");
-
+ABSL_FLAG(
+    bool, print_rules_file, false,
+    "Print the default lint values in a format that can be used to create a"
+    ".rules.verible_lint file.");
```

Comment:
To be consistent what we write in `generate_markdown` and `help_rules`, we should probably also mention the exit immediately part.

Reply:
No worries, Done.